projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package.json

```diff
@@ -59,7 +60,8 @@
     "@jupyterlab/application": "^3.1.0",
     "@jupyterlab/coreutils": "^5.1.0",
     "@jupyterlab/services": "^6.1.0",
-    "@jupyterlab/settingregistry": "^3.1.0"
+    "@jupyterlab/settingregistry": "^3.1.0",
+    "py-to-ts-interfaces": "git+https://github.com/Syndallic/py-to-ts-interfaces#main"
```

Comment:
This seems a bit dangerous, what if the main branch breaks for some reason?

Maybe you can remove this and just do something like

```
pip install py-to-ts-interfaces && npx py-to-ts-interfaces vdk-jupyterlab-extension/vdk_options/ src/vdkOptions
```
It's not ideal, but seems less hacky to me.

Reply:
changed , tho without npx since it produces error